trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -1,7 +1,7 @@
 # Trace Context HTTP Header Format
 
-A trace context header is used to pass trace context information across systems for a HTTP 
-request. Our goal is to share this with the community so that various tracing and diagnostics 
+A trace context header is used to pass trace context information across systems for an HTTP 
```

Comment:
I would avoid providing justifications here to remove redundancy. The purpose of the header is already explained in the common section. Enough to say that this part defines the format of the header specifically for HTTP protocol.